tests/mock_network/utils.rs

```diff
@@ -617,7 +617,7 @@ pub fn verify_section_invariants_for_node(node: &TestNode, elder_size: usize) {
 
     if !our_prefix.is_empty() {
         assert!(
-            our_section_elders.len() >= elder_size,
+            check_min_elder_size(our_section_elders.len(), elder_size),
```

Comment:
Ok, I think it would be good to explain in commit comment why we change that, why would we loose soo many nodes we have smaller elders. All tests are suppose to make this not happen, and in real life, with many adults this is really not suppose to be possible.

Reply:
I'll add an explanation to the commit message.